moped-editor/src/views/projects/projectView/ProjectStatusBadge.js

```diff
@@ -206,9 +206,6 @@ const ProjectStatusBadge = ({
   const chipClasses = useChipStyles(statusProperties);
   const iconClasses = useFontColorStyles(statusProperties);
 
-  // If we don't have a status, then do not render.
-  if (!!!status) return null;
-
```

Comment:
I was going to ask why remove this, but now I'm questioning why it was here to begin with. Is there a scenario where the status would ever be undefined or null? in any case, i tested setting the status to be false and the component still works fine 👍🏼

Reply:
I was wondering the same thing! From my understanding of the code, a badge will only render if certain requirements are met, so throwing this logical statement to not render under certain conditions in at the beginning seemed unnecessary and like it would cause problems. When I removed it, everything worked as expected!